### DISCUSSION TITLE
Remove ring dependencies

### DISCUSSION THREAD

**Discussion by jsiedentop** [dartclub/turf_dart#168] [Ideas]
`turf_dart`  depends on the following dartclub libraries
- turf_equality
- turf_pip
- sweepline_intersections

These libraries themselves depend on `turf_dart`. This makes the release of new versions unnecessarily complicated. 

All these libraries only depend on a really small part of `turf_dart`. It wouldn't take much effort to extract this part into a separate library and the effort would probably pay for itself very quickly.

Mainly only the GeoJson objects (`GeoJSONObject`, `GeometryCollection`, `FeatureCollection`, `Feature`, `Point`, `Position`, `MultiPoint`, `LineString`, `MultiLineString`, `MultiPolygon`, `Polygon`) are referenced. Only `turf_equality` uses another functionality: `round` from `import 'package:turf/helpers.dart';`.

It would certainly be useful to have a package that only takes over the serialization and deserialization of GeoJson in addition to the `turf_dart` packages.

@lukas-h any remarks?

**Comment by lukas-h** [comment]
Important topic.

Idea:
- let's create a package called `geojson_serialization` (or something similar) that just contains the object model
- let's just copy the code over from the helpers package, which definitely isn't elegant, but a whole dependency for just this small thing seems unnecessary. Or maybe we can have a round method inside our GeoJSON objects as a member function.

**Comment by jsiedentop** [comment]
🚀 [geotypes](https://pub.dev/packages/geotypes) package has been published.
